readable.js

```diff
@@ -1,9 +1,10 @@
-(function (){
+var Stream = (function (){
   try {
-    exports.Stream = require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
+    return require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
   } catch(_){}
 }());
 exports = module.exports = require('./lib/_stream_readable.js');
+exports.Stream = Stream || exports;
```

Comment:
All good,  just struck me as odd at first, but you're indeed right.  This is exactly mimicking the node export.